src/CSSBrush.ts

```diff
@@ -3,17 +3,18 @@ import { RGB, createColorsFromMap, rgbHex } from 'color-map'
 import { rainbow } from './colors'
 import { BrushOption, Brush } from './interfaces'
 
+const gammaMap = {
```

Comment:
yes, since it is static and won't change. Don't need to create copies of it for each brush.